lib/projects/localDev/AppDevModeInterface.ts

```diff
@@ -0,0 +1,193 @@
+import { fetchAppInstallationData } from '@hubspot/local-dev-lib/api/localDevAuth';
```

Comment:
This is a wrapper around the `UIEDevModeInterface` and includes all app-specific logic that previously lived in `LocalDevManager`. This lets us keep `LocalDevProcess` component agnostic